content/shmem_test.tex

```diff
@@ -0,0 +1,53 @@
+\apisummary{
+  Test whether a variable on the local \ac{PE} has changed.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+int shmem_test(TYPE *ivar, shmem_cmp cmp, TYPE value);
+\end{C11synopsis}
+where \TYPE{} is one of the point-to-point synchronization types specified by
+Table \ref{p2psynctypes}.
+
+\begin{Csynopsis}
+int shmem_<TYPENAME>_test(TYPE *ivar, shmem_cmp cmp, TYPE value);
+\end{Csynopsis}
+where \TYPE{} is one of the point-to-point synchronization types and has a
+corresponding \TYPENAME{} specified by Table \ref{p2psynctypes}.
+
+\begin{apiarguments}
+
+  \apiargument{OUT}{ivar}{A pointer to a remotely accessible data object.}
+  \apiargument{IN}{cmp}{The comparison operator that compares \VAR{ivar} with
+    \VAR{value}.}
+  \apiargument{IN}{value}{The value against which the object pointed to
+    by \VAR{ivar} will be compared.}
+
+\end{apiarguments}
+
+\apidescription{
+  \FUNC{shmem\_test} tests the numeric comparison of the symmetric object
+  pointed to by \VAR{ivar} with the value \VAR{value} according to the
+  comparison operator \VAR{cmp}.
+}
+
+\apireturnvalues{
+  \FUNC{shmem\_test} returns 1 if the comparison of the symmetric object
+  pointed to by \VAR{ivar} with the value \VAR{value} according to the
+  comparison operator \VAR{cmp} evalutes to true; otherwise, it returns 0.
+}
+
+\apinotes{
```

Comment:
That text was lifted from `shmem_wait[_until]()` and added to the [introductory text](https://github.com/openshmem-org/specification/pull/32/files/73fb59987c8501b243080bfcdf11f26cfaaae6df#diff-b039d6c2bf9af7510d362563dc21966aR4) to cover all the point-to-point synchronization routines.